lib/tasks/valid_test_data_generators/applications_populater.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require "tasks/valid_test_data_generators/school_urn_generator"
+require "tasks/valid_test_data_generators/trn_generator"
+
+module ValidTestDataGenerators
```

Comment:
discussed in person, maybe we can move this to services instead, same with statement